.env.development

```diff
@@ -1 +1,4 @@
 VITE_ZONE_CREATION_FLOW=enabled
+# The following are templating placeholders for the index.html file. They’re going to be replaced using server-side templating when serving the GUI application in production.
+VITE_BASE_GUI_PATH=/gui
+VITE_KUMA_CONFIG='{"baseGuiPath":"/gui","apiUrl":"http://localhost:5681","version":"2.4.0","product":"Kuma","mode":"global","environment":"universal","storeType":"postgres","apiReadOnly":false}'
```

Comment:
Note: Not keen on this, but fine by me for now (not a blocker)

Reply:
I’m not a huge fan of this either. It’s a good enough stop gap for what we need, but a bit more flexibility would be nice. Perhaps one of the plugins we used for this purpose becomes viable again. We can always switch back to them, I think.